modules/ROOT/pages/index.adoc

```diff
@@ -16,8 +16,6 @@ Refer to the DataStax Luna Streaming https://github.com/datastax/release-notes/b
 
 In addition to the distribution of https://pulsar.apache.org/en/versions/[Apache Pulsar {pulsar-version}], DataStax Luna Streaming provides:
 
-* A replicated.io package installer that can stand up a dev or production cluster on bare metal or VMs without a pre-existing Kubernetes environment
-
 * A helm chart that can deploy and manage Pulsar on your current CNCF conformant Kubernetes infrastructure
 
 * Cassandra, Elastic, Kinesis, Kafka, and JDBC connectors
```

Comment:
This not a full list. Are we planning to provide a connenctor oriented document space? If so, we can add a link here.